src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1937,29 +1966,45 @@ private void printDevModeMessages(boolean inputUnavailable, boolean startup) {
         }
         if (startup) {
             if (container) {
+                boolean nonDefaultHttpPortUsed = !skipDefaultPorts && !String.valueOf(LIBERTY_DEFAULT_HTTP_PORT).equals(httpPort);
+                boolean nonDefaultHttpsPortUsed = !skipDefaultPorts && !String.valueOf(LIBERTY_DEFAULT_HTTPS_PORT).equals(httpsPort);
+                boolean nonDefaultDebugPortUsed = alternativeDebugPort != -1; // this is set when a random ephemeral port is selected
                 if (containerHttpPort != null || containerHttpsPort != null || libertyDebug) {
                     info(formatAttentionMessage(""));
                     info(formatAttentionTitle("Liberty container port information:"));
                 }
+                if (nonDefaultHttpPortUsed || nonDefaultHttpsPortUsed || nonDefaultDebugPortUsed) {
```

Comment:
This is fine with the way the variables are currently set up, but as a precaution (against possible bugs if we change the code later on), it would be safer to also check if libertyDebug is enabled.  e.g.
```
if (nonDefaultHttpPortUsed || nonDefaultHttpsPortUsed || (libertyDebug && nonDefaultDebugPortUsed)) {
```

Reply:
😎👍